CHANGELOG.md

```diff
@@ -12,7 +12,11 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [dacaiguoguogmail](https://github.com/dacaiguoguogmail)
   [#8461](https://github.com/CocoaPods/CocoaPods/issues/8461)
 
-* Set the path of development pod groups to root directory of the Pod 
+* Adds support for referring to other podspecs during validation  
```

Comment:
nit: needs to move to updated master section after 1.7.0.beta.1

Reply:
👍  cool, yeah, done

Reply:
@orta is this an enhancement or bug fix?

Reply:
Moved it to enhancement